### ISSUE TITLE
Starting Activity in Fragments uses underlaying Activity to start the Activity, onActivityResult is not called.

### ISSUE THREAD

**Issue by matoushybl** [Kotlin/anko#199]
Hi,
I am trying to use startActivityForResult in Fragment, however the result method is not called, because the method from Anko does not start the Activity using fragment's startActivityForResult method but using Activity's.

```
inline fun <reified T: Activity> Fragment.startActivityForResult(requestCode: Int, vararg params: Pair<String, Any>) {
    AnkoInternals.internalStartActivityForResult(activity, T::class.java, requestCode, params)
}
```

Problem description I found on SO: http://stackoverflow.com/questions/6147884/onactivityresult-not-being-called-in-fragment

Could you please have a look at it?

OT: is there a method for creating a bundle from Pairs?

**Comment by yanex** [comment]
This bug was fixed in Anko 0.9, please see #76.

**Comment by yanex** [comment]
@matoushybl About making `Bundle` from pairs, please see the `bundleOf()` function (https://github.com/Kotlin/anko/blob/2af6ff83d414f78f3f5195c98b7785f29444fcd9/anko/library/static/common/src/ContextUtils.kt#L78).
